pandemia-fi/src/pages/landing/landing.tsx

```diff
@@ -1,11 +1,43 @@
-import React from "react";
+import React, { useState, useEffect } from "react";
 import Card from "../../components/card";
 import HeroChartCard from "./components/hero-chart-card";
 import NumberMetricCard from "./components/number-metric-card";
 import LinksCard from "./components/links-card";
 import "./landing.css";
 
-const LandingPage: React.FunctionComponent = () => {
+interface CaseEntry {
+  id: number;
+  date: string;
+  healthCareDistrict: string;
+}
+
+function LandingPage() {
+
+  const [numsConfirmed, setNumsConfirmed] = useState(0);
+  const [numsConfirmedToday, setNumsConfirmedToday] = useState(0);
+  const [numsRecovered, setNumsRecovered] = useState(0);
+  // const [numsDeaths, setNumsDeaths] = useState(0);
```

Comment:
Seeing this here nudged me to think that showing the amount of deaths might be good for the dashboard. However, that's a concern for another PR, no need to update anything here.